tests/telemetry_derived/clients_histogram_probe_counts_v1/test_aggregation/expect.ndjson

```diff
@@ -0,0 +1,6 @@
+{"agg_type":"histogram","aggregates":[{"key":"0","value":0.2},{"key":"1","value":0},{"key":"2","value":0.1},{"key":"3","value":0},{"key":"5","value":0.2},{"key":"9","value":0.3},{"key":"16","value":0},{"key":"29","value":0},{"key":"54","value":0.2},{"key":"100","value":0}],"app_build_id":"first","app_version":75,"channel":"nightly","client_agg_type":"summed_histogram","key":"","metric":"test","metric_type":"histogram-exponential","os":"first","process":"parent","total_users":1}
+{"agg_type":"histogram","aggregates":[{"key":"0","value":0.2},{"key":"1","value":0},{"key":"2","value":0.1},{"key":"3","value":0},{"key":"5","value":0.2},{"key":"9","value":0.3},{"key":"16","value":0},{"key":"29","value":0},{"key":"54","value":0.2},{"key":"100","value":0}],"app_build_id":"first","app_version":75,"channel":"nightly","client_agg_type":"summed_histogram","key":"","metric":"test","metric_type":"histogram-exponential","process":"parent","total_users":1}
+{"agg_type":"histogram","aggregates":[{"key":"0","value":0.2},{"key":"1","value":0},{"key":"2","value":0.1},{"key":"3","value":0},{"key":"5","value":0.2},{"key":"9","value":0.3},{"key":"16","value":0},{"key":"29","value":0},{"key":"54","value":0.2},{"key":"100","value":0}],"app_build_id":"second","app_version":75,"channel":"nightly","client_agg_type":"summed_histogram","key":"","metric":"test","metric_type":"histogram-exponential","os":"first","process":"parent","total_users":1}
+{"agg_type":"histogram","aggregates":[{"key":"0","value":0.2},{"key":"1","value":0},{"key":"2","value":0.1},{"key":"3","value":0},{"key":"5","value":0.2},{"key":"9","value":0.3},{"key":"16","value":0},{"key":"29","value":0},{"key":"54","value":0.2},{"key":"100","value":0}],"app_build_id":"second","app_version":75,"channel":"nightly","client_agg_type":"summed_histogram","key":"","metric":"test","metric_type":"histogram-exponential","process":"parent","total_users":1}
+{"agg_type":"histogram","aggregates":[{"key":"0","value":0.4},{"key":"1","value":0},{"key":"2","value":0.2},{"key":"3","value":0},{"key":"5","value":0.4},{"key":"9","value":0.6},{"key":"16","value":0},{"key":"29","value":0},{"key":"54","value":0.4},{"key":"100","value":0}],"app_version":75,"channel":"nightly","client_agg_type":"summed_histogram","key":"","metric":"test","metric_type":"histogram-exponential","os":"first","process":"parent","total_users":2}
```

Comment:
note that `total_users` is 2 here since there is no `app_build_id` and the input yaml is for a single client appearing once for each `app_build_id` they submitted